packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -281,6 +286,9 @@ class MessageList extends Widget<Properties> {
 
       if (items.length - 1 === index) {
         this._createMessageGroupComponent(currentMessageGroupItems, currentMessageGroupUserId);
+
+        const lastMessageGroup = this._messageGroups?.at(-1);
+        lastMessageGroup?._updateIsLastOptionOfLastMessage(true);
```

Comment:
[excess template call]
That's not good that we we render last bubble twice: with `isLast=false` and then with `isLast=true`

Reply:
I thought we decided we don't want to add `isLast` option in `messageGroup`. This means we cannot pass correct `isLast` option to `messageBubble` on first render.

Reply:
Added `isLast` on `messageGroup` lvl, now there is no excess rerender 0e11dd012baad48fce62eeab905c60abc39aa9d5